fenix/app/src/main/res/values-night/colors.xml

```diff
@@ -33,6 +41,14 @@
     <color name="fx_mobile_action_color_tertiary" tools:ignore="UnusedResources">@color/photonDarkGrey10</color>
     <!-- Chip -->
     <color name="fx_mobile_action_quarternary" tools:ignore="UnusedResources">@color/photonDarkGrey80</color>
+    <!-- Warning button -->
+    <color name="fx_mobile_action_warning" tools:ignore="UnusedResources">@color/photonYellow40A41</color>
```

Comment:
```suggestion
    <color name="fx_mobile_action_color_warning" tools:ignore="UnusedResources">@color/photonYellow40A41</color>
```
Just aligning this with the token name that is listed in the stickersheet. We will need to do the same for the other button colors below.

Could you also correct the `fx_mobile_action_quarternary` above while we're at it?